src/index.html

```diff
@@ -1,12 +1,328 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <img
+        class="logo"
+        src="../src/images/logo.png"
+        alt="Moyo logo"
+      >
```

Comment:
Consider wrapping the logo into the link

Reply:
Not fixed

Reply:
I added block 'img' in tag a. Why is that wrong?